iroh-relay/src/client.rs

```diff
@@ -903,19 +525,17 @@ impl Actor {
             TcpStream::connect(proxy_addr).await
         })
         .await
-        .map_err(|_| ClientError::ConnectTimeout)?
-        .map_err(ClientError::DialIO)?;
+        .context("Timeout connecting")?
+        .context("Error connecting")?;
```

Comment:
no need to write `error` in these contexts, these are already shown as errors

Reply:
Fair, I probably did that because of the "timeout" clarification in the other one.